package.json

```diff
@@ -33,7 +35,16 @@
     "devDependencies": {
         "@babel/cli": "^7.10.1",
         "@babel/core": "^7.10.1",
-        "@babel/preset-env": "^7.10.1"
+        "@babel/preset-env": "^7.10.1",
+        "@babel/preset-react": "^7.12.1",
+        "@cypress/code-coverage": "^3.8.2",
+        "babel-loader": "^8.1.0",
+        "babel-plugin-istanbul": "^6.0.0",
+        "cypress": "^5.5.0",
+        "cypress-react-unit-test": "^4.16.7",
+        "react": "^16.13.1",
```

Comment:
@AleksandarDev should we have `react` and `react-dom` as `devDependencies` or as `peerDependencies`?

Reply:
dev, used only for running tests

Reply:
Hooks rely on it as well.

Reply:
Hooks will use react installed as peerdep

Reply:
Makes sense.